Tasks/PublishSymbolsV2/PublishSymbols.ts

```diff
@@ -7,6 +7,12 @@ import * as clientToolRunner from "azure-pipelines-tasks-packaging-common/univer
 import * as tl from "azure-pipelines-task-lib/task";
 import { IExecSyncResult, IExecOptions } from "azure-pipelines-task-lib/toolrunner";
 
+const nodeVersion = parseInt(process.version.split('.')[0].replace('v', ''));
+if(nodeVersion < 16) {
+    console.warn("Support for symbolpublish task with node version less than 16 is ending at 07/31/2024. Please upgrade your agent to node 16 and above to avoid any failures in pipeline related to the task.");
```

Comment:
please use tl.warning for warnings, otherwise they won't be displayed on AzDevOps pipeline page ([example](https://github.com/microsoft/azure-pipelines-tasks/blob/master/Tasks/GradleV2/gradletask.ts#L185)) 

Also if possible use localization